### DISCUSSION TITLE
Big gap of AP50 scores for Faster RCNN run on MMDetection and detectron2

### DISCUSSION THREAD

**Discussion by hanswang1** [open-mmlab/mmdetection#4795] [Q&A]
I have run Faster RCNN model on both MMDetection and detectron2 frame.
The hardware platform is the same sever with Intel 940X CPU and 2 GeForce 3080 GPUs for these 2 runs.
The training and validate set are same for the 2 runs (12570 pictures)
The test set is also same for these 2 runs (274 pictures)

The AP50 scores, batch size and base learning rate are as below:
Model                                                                      | AP50 | mAP | Epoch | Batch Size | Base Learning Rate
MMDetection Faster RCNN (With  Augmentation) | 68.8  | NA    | 80       | 8               | 0.003
Detectron2 Faster RCNN (With  Augmentation)     | 81.2  | 56.9   | 74      | 12             | 0.01

I am quite confuse about the big gap of AP50 scores for Faster RCNN run on MMDetection and detectron2 framework.
Does any one has any idea about the difference between MMDetection and detectron2  for Faster RCNN?

**Comment by abfleishman** [comment]
I am running Mask RCNN models and finding something similar with the detectron2 model performing much better (in the numeric output and when I look at the predicted masks).  I would love to hear if you got to the bottom of this?

**Reply by hanswang1** [reply]
The answer is found.
Detectron2 integrated Multi-scale Automatically, While MM-Detection did not include multi-scale automatically.
